packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -288,7 +291,11 @@ export class ContainerDevtools implements IContainerDevtools, HasContainerKey {
 		[DisconnectContainer.MessageType]: async (untypedMessage) => {
 			const message = untypedMessage as DisconnectContainer.Message;
 			if (message.data.containerKey === this.containerKey) {
+				// Once disconnected, the devtools user will no longer be able to operate the container, it should be removed from the audience list.
+				const audience = this.container.audience as IAudienceOwner;
+				audience.removeMember(this.container.clientId as string);
 				this.container.disconnect(/* TODO: Specify devtools reason here once it is supported */);
+				this.postContainerStateChange();
```

Comment:
Should this be added to the `[CloseContainer.MessageType]` ?

Reply:
Should be but I'm also wondering should all members be removed after close a container?